packages/core/src/routes/interaction/utils/single-sign-on-guard.ts

```diff
@@ -55,7 +55,7 @@ export const verifySsoOnlyEmailIdentifier = async (
  * @param ctx
  * @param provider
  * @param connectorId
- * @returns The single sign on session data
+ * @returns SingleSignOnConnectorSession
```

Comment:
these are not needed since we already have typesciprt

Reply:
removed